log4j-mongodb4/pom.xml

```diff
@@ -88,6 +88,17 @@
       <artifactId>de.flapdoodle.embed.mongo</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-slf4j-impl</artifactId>
```

Comment:
The test is about using slf4j on top of log4j2 with mongodb4 appender. By using slf4j without adding that dependency, the log message (Hello log) will not get written to the mongodb database so the test would fail. Also there are warning message being shown :
`SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".`
`SLF4J: Defaulting to no-operation (NOP) logger implementation`
`SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.`
Do i need to remove that added dependency or something?

Reply:
Can you reproduce the issue by using the log4j API? IOW, without going through the slf4j layer? If this is an issue in log4j, adding slf4j should not be needed to show the issue.